src/components/popup/details/purposes/purposes.jsx

```diff
@@ -133,11 +141,17 @@ export default class Purposes extends Component {
 			(isPublisher ? selectedPublisherPurposeIds : selectedPurposeIds).has(purpose.id)
 		);
 
+		const isPublisherLegitimateInterestActive = (id) => selectedPublisherLegitimateInterests.has(id);
+		const isPurposeLegitimateInterestActive = (id) => selectedPurposeLegitimateInterests.has(id+1);
+
 		const purposeIsTechnical = (purpose) => config.legIntPurposeIds &&
 			config.contractPurposeIds &&
 			purpose && !purpose.custom &&
 			(config.legIntPurposeIds.indexOf(purpose.id) >= 0 || config.contractPurposeIds.indexOf(purpose.id) >= 0);
 
+		const selectedSpecialPurposes = config.specialPurposes && !!config.specialPurposes.length &&
```

Comment:
Can we change the name of this variable: i.e: publisherSpecialPurposes ? It is confusing since special purposes are not selectable.